grpc-interface-support/test/integration/test_integration_python.py

```diff
@@ -87,15 +85,15 @@ def test_pip_package_is_usable():
 
     assert subprocess.check_call(["velocitas", "init", "-v"]) == 0
 
-    launcher = start_app("launcher", envs)
-
-    assert launcher.returncode == 0
+    server_process = start_app("launcher", envs)
 
     print("============= TEST CLIENTS ===================")
     os.chdir(os.environ["SERVICE_CLIENT_ROOT"])
 
     assert subprocess.check_call(["velocitas", "init", "-v"]) == 0
 
-    launcher = start_app("launcher", envs)
+    client_process = start_app("launcher", envs)
 
-    assert launcher.returncode == 0
+    client_code = client_process.wait()
+    server_process.kill()
```

Comment:
Since we are not gracefully shutting down the server, the return code is most likely going to be != 0. Let's tidy up Python after this and create a follow-up ticket.

Reply:
We decided to tackle this with a follow-up ticket together with the python sdk fix